### DISCUSSION TITLE
Alteração de caminho diretamenta no sleuth.db

### DISCUSSION THREAD

**Discussion by eniocc** [sepinf-inc/IPED#2002] [Q&A] [answered]
Processei uma evidência e esqueci de colocar que o processamento deveria considerar o path relativo.
Quais tabelas devo alterar o caminho dos arquivos .E01 além da tabela tsk_image_names?

Abri o sleuth.db e utilizei o seguinte comando para apontart para o caminho relativo

```
UPDATE tsk_image_names
SET path = REPLACE(path, 
                   'Z:\item-01\', 
                   '..\item-01\');

```

Mesmo após essa alteração o exe não carrega.

Alguma sugestão?

**Comment by lfcnassif** [comment, marked as answer]
If I remember, it is just `tsk_image_names` table. I would suggest you to revert your manual changes, since the application searches for the evidence in a couple of places if it is not found in its original location and test if things continue working if you change the drive letter:
https://github.com/sepinf-inc/IPED/wiki/User-Manual#--portable

**Reply by eniocc** [reply]
Thanks.
